alphadia/outputtransform.py

```diff
@@ -653,9 +654,13 @@ def build_stat_df(
         stat_df_list = []
         for folder in folder_list:
             raw_name = os.path.basename(folder)
+            optimization_manager_path = os.path.join(folder, "optimization_manager.pkl")
```

Comment:
could we instead instantiate the `OptimizationManager(load_from_file=True)` here? We will most likely switch away from pickle at some point and then we only have the functionality for storing/loading within the `OptimizationManager`.

Reply:
That sounds like a good idea. I will change it.